src/TmsIntegrationExample.Services/TmsGatewayApi/Model/AccessorialCode.cs

```diff
@@ -0,0 +1,91 @@
+﻿// <copyright file="AccessorialCode.cs" company="Engaged Technologies">
+//      Copyright (c) Engaged Technologies. All rights reserved.
+//  </copyright>
+// <summary>
+//     Accessorial Code possible values
+// </summary>
+
+namespace TmsIntegrationExample.Services.TmsGatewayApi.Model
+{
+    /// <summary>
+    /// An enumeration of possible accessorial codes for customer requested accessorials. If anything other than a customer request item is put in this enumeration Joy will come after you with fire and fury.
+    /// </summary>
+    // THE ORDER MATTERS! These enums are relating to ints that are stored in our DB so how these map back from the DB to enums has been defined and reordering is not allowed.If you need to add a new state do it at the end please :)
```

Comment:
I would definitely prefer that we don't include this. Yes, it matters to us, but we should not even allow TmsGateway clients to use integers for enum values when calling the api so to clients it shouldn't matter.

Reply:
do we need to remove the comments for all models classes or just this one?

Reply:
All of them